cmd/scorecard_option.go

```diff
@@ -23,7 +23,7 @@ func ValidateScorecardOption(opt string) error {
 }
 
 func IsScorecardEnabled(when string) bool {
-	return when != scNo
+	return when == scYes || when == scVerbose
```

Comment:
I'm not sure this is related to this pr but anyway, why would running in verbose  mode imply that we want to get scorecard?

Reply:
both flags relate to scorecard (sc == scorecard) and not in general to the verbosity of the program